src/ir/static_init.rs

```diff
@@ -112,11 +112,9 @@ impl<B: Backend> Compiler<B> {
             })
         })
     }
-    pub(super) fn compile_string(
-        &mut self,
-        string: Vec<u8>,
-        location: Location,
-    ) -> CompileResult<DataId> {
+    pub(super) fn compile_string(&mut self, location: Location) -> CompileResult<DataId> {
+        let string = self.resolve_location(location);
+        let string: Vec<u8> = string.get(1..string.len() - 1).unwrap().as_bytes().to_vec(); // Remove quotation marks // TODO escapes?
```

Comment:
RE escapes: this seems like something we might have to calculate in various different areas. Instead of redoing it every time, we could have a query infrastructure like rustc using [salsa](https://docs.rs/salsa/0.14.1/salsa/). That also gets us incremental compilation almost for free.

cc @pythondude325